### ISSUE TITLE
AssertionError when calling env.step() in prison_v0 environment with a continuous action space

### ISSUE THREAD

**Issue by dgauraang** [PettingZoo-Team/PettingZoo#160]
Error:

> Action: [1.8698319]
> Action: [-0.04900217]
> Action: [-2.125496]
> Action: [0.01161803]
> Action: [0.27126586]
> Action: [0.41918978]
> Action: [-1.5011265]
> Action: [2.418947]
> Traceback (most recent call last):
>   File "test_pettingzoo.py", line 19, in <module>
>     observation = env.step(action)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 306, in step
>     return super().step(action, observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 95, in step
>     super().step(action, False)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 60, in step
>     next_obs = self.env.step(action, observe=observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 177, in step
>     return super().step(action, observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 60, in step
>     next_obs = self.env.step(action, observe=observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 252, in step
>     return super().step(action, observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/utils/wrappers.py", line 60, in step
>     next_obs = self.env.step(action, observe=observe)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/butterfly/prison/prison.py", line 334, in step
>     self.draw()
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/butterfly/prison/prison.py", line 267, in draw
>     self.screen.blit(self.prisoners[p].get_sprite(), self.prisoners[p].position)
>   File "/home/gauraang/.local/lib/python3.6/site-packages/pettingzoo/butterfly/prison/prison.py", line 59, in get_sprite
>     assert False, ("INVALID STATE", self.state)
> AssertionError: ('INVALID STATE', array([1.], dtype=float32))
> 

Code to reproduce the error:

```
import time
from pettingzoo.butterfly import prison_v0
env = prison_v0.env(seed=1, continuous=True)

observation = env.reset()
for agent in env.agent_iter():
    reward, agent_done, info = env.last()
    action = env.action_spaces[agent].sample()
    print('Action:', action)
    observation = env.step(action)

    env.render()

```

**Comment by justinkterry** [comment]
Tests should've caught this, we'll fix it very soon

**Comment by dgauraang** [comment]
EDIT:
I didn't have any issues when I changed the code by hardcoding action as 1.

```
import time
from pettingzoo.butterfly import prison_v0
env = prison_v0.env(seed=1, continuous=True)

observation = env.reset()
for agent in env.agent_iter():
    reward, agent_done, info = env.last()
    action = env.action_spaces[agent].sample()
    
    observation = env.step(1)

    env.render()

```
Should the action be clipped if an agent is approaching the boundary?

Thanks for the quick response by the way!

**Comment by weepingwillowben** [comment]
Should be solved by #161.

Looks like my parameter tests simply weren't running correctly. Issues like this should be much more rare in the future now that there are proper tests for it.

**Comment by justinkterry** [comment]
Fixed released (with version 1.0.1), sorry about that
